tests/acceptance/test_bootstrap_artifact.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/python
+# Copyright 2022 Northern.tech AS
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import pytest
+import os
+import time
+
+from conftest import setup_qemu
+
+
+from utils.common import (
+    build_image,
+    signing_key,
+    reset_build_conf,
+)
+
+
+@pytest.fixture(scope="function")
+def build_image_with_signed_bootstrap_artifact(
```

Comment:
Also copy-pasta from the `mender-image-test` fixtures, with one notable difference.

These are `function` scoped, and they provide a different set of build-variables.

Feel it's okay, since it's only used here locally.

Otherwise, we still have to duplicate the functions somewhere, as I found no good way of _borrowing_ a fixture, and change it's scope... But feel free to take me to school here.

Reply:
After f2f: Sort it first and comment on it in here.